docs/ssm-on-aws/ssm-on-aws.md

```diff
@@ -0,0 +1,374 @@
+                    "func": "time_http_call",
+                    "arguments": {
+                        "url": "${endpoint_url}"
+                    }
+                },
+                "tolerance": {
+                    "name": "http-response-time-under",
+                    "type": "probe",
+                    "provider": {
+                        "type": "python",
+                        "module": "chaoscloud.tolerances.http",
+                        "func": "response_time_under",
+                        "arguments": {
+                            "duration": 6000
+                        }
+                    }
+                }
+            }
+        ]
+    },
+    "method": [
+        {
+            "type": "action",
+            "name": "inject-cpu",
+            "provider": {
+                "type": "process",
+                "path": "aws",
+                "arguments": "ssm send-command --document-name \"cpu-stress\" --document-version \"1\"
+--targets '[{\"Key\":\"InstanceIds\",\"Values\":[\"${instance_id}\"]}]'
+--parameters '{\"duration\":[\"180\"],\"cpu\":[\"0\"]}' --timeout-seconds 60
+--max-concurrency \"50\" --max-errors \"0\" --output-s3-bucket-name \"gtf-chaos-output\"
+--cloud-watch-output-config '{\"CloudWatchOutputEnabled\":true}'
+--region eu-west-1 --query 'Command.CommandId' > CommandId.info"
+            }
+        }
+    ],
+    "rollbacks": [
+        {
+            "type": "action",
+            "name": "Stop SSM run command",
+            "provider": {
+                "type": "process",
+                "path": "aws",
+                "arguments": "ssm cancel-command --command-id $(cat CommandId.info) "
+            },
+            "pauses": {
+                "after": 10
+            }
+        }
+    ]
+}
+```
+
+!!!Note
+    You will need to add your own S3bucket name.
+
+
+It worth connecting to your EC2 instance now using [SSH][SshAccessToEc2] and running ```htop``` so you can see the impact of the SSM command on the CPU.
+
+The experiment can be run with a  command similar to the following, but you should replace the ```ENDPOINT_URL``` and the ```INSTANCE_ID``` with your own EC2 values:
+
+```bash
+ENDPOINT_URL=http://ec2-52-18-8-61.eu-west-1.compute.amazonaws.com/random/4 \
+INSTANCE_ID=i-086ccae1f2de9b012 \
+chaos run verification-ec2-responds-processor-load-stage.json
+```
+
+Shortly after the experiment begins the ```htop``` session should look similar to:
+
+![htop screenshot][HtopScreen]
+
+At this point, some stress has been applied to the CPU and a Steady
+ State Hypothesis has run against the EC2 endpoint. An experiment only applies
+ the Steady State Hypothesis at the beginning and at the end of an experiment,
+  it would be better to run it as a verification where the Steady State
+  Hypothesis can be repeated over a period of time.
+
+## Experiment Import
+
+The ChaosIQ Reliability Toolkit supports the import of an experiment to into
```

Comment:
The Reliability Toolkit

I don't think we have to keep reminding them of the company behind it? Just my take, wdyt @Lawouach ?